.prettierrc.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Not that Prettier is required, but [here are](https://github.com/texastribune/texastribune/blob/master/package.json#L165-L171) the settings we use in `texastribune`, if that's helpful.

Reply:
@AndrewGibson27 Added this and ran prettier. Very helpful :)